salt/grains/napalm.py

```diff
@@ -73,8 +73,7 @@ def _retrieve_grains_cache(proxy=None):
             # if proxy var not passed and is running in a straight minion
             GRAINS_CACHE = salt.utils.napalm.call(
                 DEVICE_CACHE,
-                'get_facts',
-                **{}
+                'get_facts'
```

Comment:
You should go ahead and leave the trailing comma - it's not a syntax error in Python, and if another author comes in and adds more arguments, you'll properly be credited for `'get_facts',`, instead of the next author getting blamed for that line when all they did was add a `,` 🙃 

Otherwise 👍